ome/omero-install
# Apache drop

Remove apache from omero-install

To test:
 * Make sure that there is no ref to apache
 * travis should be green
 * generate the walkthrough (``i.e. bash autogenerate.sh``) and check that there is no mention of apache in the files
